cmd/cluster-init/update_jobs.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/jobconfig"
+)
+
+type InfraPeriodics struct {
+	Periodics []prowconfig.Periodic `json:"periodics,omitempty"`
+}
+
+func updateJobs(o options) error {
+	var config prowconfig.JobConfig
+	updatePresubmits(o, &config)
+	updatePostsubmits(o, &config)
+	updateInfraPeriodics(o, &config)
+
+	metadata := RepoMetadata()
+	return jobconfig.WriteToDir(filepath.Join(o.releaseRepo, "ci-operator", "jobs"), metadata.Org, metadata.Repo, &config)
+}
+
+func updatePresubmits(o options, config *prowconfig.JobConfig) {
```

Comment:
Two nits: first, this is not really updating anything, second, if we're calling this but once, do we need a standalone function? Why not

```go
func updateJobs(o options) error {
    return jobconfig.WriteToDir(filepath.Join(o.releaseRepo, "ci-operator", "jobs"), "openshift", "release", &prowconfig.Config{
        PresubmitsStatic: map[string][]prowconfig.Presubmit{
		"openshift/release": {generatePresubmit(o.clusterName)},
	},
	// ...
    })
}
}
```

Reply:
yeah this function has outlived its usefulness. It used to be doing a lot more. I'll inline it.